go/packages/dev-mode/agent/forwarder.go

```diff
@@ -143,6 +147,52 @@ func FindReqData(logs []LogItem) *LogItem {
 	return nil
 }
 
+func IsDefaultConsoleMessage(record interface{}) bool {
+	message := fmt.Sprintf("%v", record)
+	arr := strings.Split(message, "\t")
+	// Not enough items in the list
+	// The first item is not a date timestamp
+	// The second item is not a request id
+	if !hasInternalExtension || len(arr) < 4 || !IsValidDate(arr[0]) || !IsValidUUID(arr[1]) {
```

Comment:
@medikoo the `hasInternalExtension` check that we have the node internal extension before filtering any logs 👍